### PR TITLE
#777 added in scholarship section

### PR DESCRIPTION
**Pull request by Kumaranupam2020**
added a scholarship program "Venkat Panchapakesan Memorial Scholarship" which is for women and non-binary engineers

### CODE REVIEW

**Comment by Kundan28** [comment]
@Kumaranupam2020 Please write descriptive title and mention the issue number in the comments.

**Comment by Kumaranupam2020** [comment]
I have added 1 line in the scholarship section under issue #777

**Comment by Kundan28** [comment]
> I have added 1 line in the scholarship section under issue #777

#777 is the PR number not the issue number.

**Comment by Kumaranupam2020** [comment]
I have added a scholarship program in scholarship section under issue number #560

**Comment by Kundan28** [comment]
@Kumaranupam2020 Was that issue assigned to you ?

File: women-in-tech-resources/resources-list.md [hunk 1]
```diff
@@ -69,6 +69,7 @@ If this is your first time looking at the Women in Tech Resources List, spend 30
 ### Scholarships:
 - [SWE Scholarship](http://societyofwomenengineers.swe.org/swe-scholarships)
 - [Palantir Scholarship](https://www.palantir.com/students/scholarship/wit-north-america/)
+-Venkat Panchapakesan Memorial Scholarship(https://buildyourfuture.withgoogle.com/scholarships/venkat-panchapakesan-memorial-scholarship/#!?detail-content-tabby_activeEl=overview)
```
Comment (Kundan28) [inline women-in-tech-resources/resources-list.md]: @Kumaranupam2020 Put `Venkat Panchapakesan Memorial Scholarship` in square bracket like this [Venkat Panchapakesan Memorial Scholarship].

**Comment by Kumaranupam2020** [comment]
Thank you

File: women-in-tech-resources/resources-list.md [hunk 2]
```diff
@@ -69,6 +69,7 @@ If this is your first time looking at the Women in Tech Resources List, spend 30
 ### Scholarships:
 - [SWE Scholarship](http://societyofwomenengineers.swe.org/swe-scholarships)
 - [Palantir Scholarship](https://www.palantir.com/students/scholarship/wit-north-america/)
+-[Venkat Panchapakesan Memorial Scholarship](https://buildyourfuture.withgoogle.com/scholarships/venkat-panchapakesan-memorial-scholarship/#!?detail-content-tabby_activeEl=overview)
```
[the remark this answers is not in the document]
Comment (Kundan28) [reply women-in-tech-resources/resources-list.md]: Yeah @Kumaranupam2020

**Pull request merged**